docs/CLI.md

```diff
@@ -37,15 +34,17 @@ Validate an off-chain Nexus Tool on the provided URL. This command checks whethe
 2. `GET /health` simple health check endpoint that needs to return a `200 OK` in order for the validation to pass.
 3. `POST /invoke` the CLI can check that the endpoint exists.
 
-> As an improvement, the command could take a `[data]` parameter that invokes the Tool and checks the response against the output schema.
+{% hint style="success" %}
+As an improvement, the command could take a `[data]` parameter that invokes the Tool and checks the response against the output schema.
+{% endhint %}
 
 This command should also check that the URL is accessible by the Leader node. It should, however, be usable with `localhost` Tools for development purposes, printing a warning.
 
 ---
 
 **`nexus tool validate --on-chain <ident>`**
 
-[Issue #96](https://github.com/Talus-Network/nexus-next/issues/96)
+🚧 This is TBD... 🚧
```

Comment:
Shouldn't we remove this instead of not documenting it? Reference: https://developers.google.com/style/future

Reply:
The counter argument is that the CLI already does reference it. So the user could be confused

Reply:
I agree with @kouks here, we should let the users know this command isn't implemented yet.

Reply:
will rephrase then

Reply:
rephrased in [4f6b9a0](https://github.com/Talus-Network/nexus-sdk/pull/136/commits/4f6b9a079f06a2740aeaaa489dbdd22346192cc3)